modules/ossm-routing-destination-rules.adoc

```diff
@@ -9,7 +9,7 @@
 
 Destination rules are applied after virtual service routing rules are evaluated, so they apply to the traffic's real destination. Virtual services route traffic to a destination. Destination rules configure what happens to traffic at that destination.
 
-By default, {SMProductName} uses a round-robin load balancing policy, where each service instance in the pool gets a request in turn. {SMProductName} also supports the following models, which you can specify in destination rules for requests to a particular service or service subset.
+By default, {SMProductName} uses a least requests load balancing policy, where each service instance in the pool gets a request in turn. {SMProductName} also supports the following models, which you can specify in destination rules for requests to a particular service or service subset.
```

Comment:
I'm looking at the resource schema and these are the allowed values: `LEAST_CONN`, `RANDOM`, `PASSTHROUGH`, `ROUND_ROBIN`, and `LEAST_REQUEST`. So, I guess this should be "least request" or maybe "least-request"?